ISSUE_TEMPLATE.md

```diff
@@ -1,14 +1,14 @@
 ***PCSX2 version:***
 ``(For example, stable build v1.4.0 (or) development builds v1.5.xxx)``
 
-- **Note**: It's preferred to only report bugs which are present on the [latest development builds](http://pcsx2.net/download/development.html).
-
-- **Note2**: The new backwards compatibility (psx) mode is in active development. Issue reports are too early at this point.
+- **Notes**:
+  - It's preferred to only report bugs which are present on the [latest development builds](http://pcsx2.net/download/development.html).
+  - The new PSX backward compatibility mode is still in its nascent state. Issue reports are too early at this point.
```

Comment:
Side note: I'm not sure that non-native English speakers or indeed, native English speakers, will be familiar with the meaning of "nascent". Personally I understood it via contextual clues at first, and then I looked it up to confirm the meaning.